### PR TITLE
Add PureComponent

### PR DESCRIPTION
**Pull request by neo**

### CODE REVIEW

File: 11.PureComponent.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+**Jam3 Lessons - React**
+
+# React.PureComponent
+
+> React.PureComponent is exactly like `React.Component` but implements `shouldComponentUpdate()` with a shallow prop and state comparison.
+
+> If your React component's `render()` function renders the same result given the same props and state, you can use `React.PureComponent` for a performance boost in some cases.
+
```
Comment (jeremenichelli) [inline 11.PureComponent.md:8]: Put content as paragraph instead of blockquotes.

Basically, remove both `>`.

File: 11.PureComponent.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+**Jam3 Lessons - React**
+
+# React.PureComponent
+
+> React.PureComponent is exactly like `React.Component` but implements `shouldComponentUpdate()` with a shallow prop and state comparison.
+
+> If your React component's `render()` function renders the same result given the same props and state, you can use `React.PureComponent` for a performance boost in some cases.
```
Comment (jeremenichelli) [inline 11.PureComponent.md:7]: I think it sounds better with this changes:

`If your component's render function renders the same result given the same props and state, you can use it as a performance boost and write less code.`

Let me know what you think @neo.

File: 11.PureComponent.md (hunk 1 above)
Comment (neo) [reply 11.PureComponent.md:8]: I used quotes because I copied it from the React docs, and I think that's the best term to describe this. @jeremenichelli

Comment (jeremenichelli) [reply 11.PureComponent.md:8]: Can we modify it a little bit so we generate a more sense of ownership over the lesson?

**Pull request merged**